utils/calculate-files-hash.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const crypto = require('crypto');
+const fileHash = require('./calculate-file-hash');
```

Comment:
Any reason why this file is requiring it self?

Reply:
umm... whoops?

Reply:
oh, no its not. `calculate-files-hash` is requiring `calculate-file-hash`. The first is plural

Reply:
Ahh..